src/Compilers/VisualBasic/Test/Syntax/Scanner/ScannerTests.vb

```diff
@@ -508,9 +508,59 @@ Public Class ScannerTests
         Assert.Equal(" _ ", tk.ToFullString())
 
         tk = ScanOnce(" _'", startStatement:=True)
-        Assert.Equal(SyntaxKind.BadToken, tk.Kind)
+        Assert.Equal(SyntaxKind.EndOfFileToken, tk.Kind)
         Assert.Equal(" _'", tk.ToFullString())
-        Assert.Equal(30999, tk.Errors.First().Code)
+        Assert.Equal(3, tk.LeadingTrivia.Count)
+        Assert.True(tk.LeadingTrivia(0).Kind = SyntaxKind.WhitespaceTrivia)
+        Assert.True(tk.LeadingTrivia(1).Kind = SyntaxKind.LineContinuationTrivia)
+        Assert.True(tk.LeadingTrivia(2).Kind = SyntaxKind.CommentTrivia)
+        Assert.True(tk.Errors.Count = 1)
```

Comment:
>Assert.True(tk.Errors.Count = 1) [](start = 8, length = 32)

```Assert.Equal```? As an alternative, you could do ```Single``` instead of ```First``` in the assert below, then this assert wouldn't bee needed.

Reply:
I don't see a Single listed as an option for SyntaxDiagnosticInfoList